### ISSUE TITLE
Information: Lost `laravel-lang/lang` on update

### ISSUE THREAD

**Issue by andrey-helldar** [Laravel-Lang/publisher#204]
## Actions

Run the `composer update`  and updated the package to `11.2.1` version.

The following message appeared in the console:

```
Removing laravel-lang/lang
```

## What happened

Starting with publisher version `11.2`, we switched to a new model, now this package is not tied to a specific project with translations, in addition, such changes reduce code duplication when testing the `laravel-lang/lang` repository.

## How to fix it

Starting from version `11.2`, you need to install the `laravel-lang/lang` dependency in your application yourself:

```bash
composer require laravel-lang/publisher laravel-lang/lang --dev
```

We have updated the documentation section about this: https://laravel-lang.github.io/publisher/installation.html

---
PS: @Laravel-Lang/laravel-lang, those of you who use this [`publisher`](https://github.com/Laravel-Lang/publisher) will also be interesting🙂

**Comment by andrey-helldar** [comment]
Q: Why were these changes included in the minor release and not the major?
A: Because.

**Comment by andrey-helldar** [comment]
We decided to fix the update issue and released patch [`11.2.3`](https://github.com/Laravel-Lang/publisher/releases/tag/v11.2.3), which brings the installer back to its previous state.

We recommend updating the publisher version to [`12.0`](https://github.com/Laravel-Lang/publisher/releases/tag/v12.0.0).
